R/qenv-get_code.R

```diff
@@ -11,7 +12,7 @@
 #' get_code(q3)
 #' get_code(q3, deparse = FALSE)
 #' @export
-setGeneric("get_code", function(object, deparse = TRUE) {
+setGeneric("get_code", function(object, deparse = TRUE, names = NULL) {
```

Comment:
This doesn't have to be an S4 generic, does it? There is only one method and it's single dispatch.

Reply:
right